lms/static/js/search/views/SearchForm.js

```diff
@@ -0,0 +1,62 @@
+RequireJS.define(['jquery', 'backbone'], function ($, Backbone) {
+   'use strict';
+
+    return Backbone.View.extend({
+
+        el: '#courseware-search-bar',
+        events: {
+            'submit form': 'submitForm',
+            'click .cancel-button': 'clearSearch',
+        },
+
+        initialize: function () {
+            this.$searchField = this.$el.find('.search-field');
+            this.$searchButton = this.$el.find('.search-button');
+            this.$cancelButton = this.$el.find('.cancel-button');
+        },
+
+        submitForm: function () {
+            this.doSearch();
+            // prevent reload
+            return false;
```

Comment:
This works, but it would be clearer to have this function accept an `event` argument, and call `event.preventDefault()`, which accomplishes the same thing.